File: reservation/src/main/java/com/unmsm/distributedsystems/reservation/model/api/Article.java
package com.unmsm.distributedsystems.reservation.model.api;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.io.Serializable;

@Builder
@Setter
@Getter
@AllArgsConstructor
public class Article implements Serializable {

    private Integer id;
    private String name;
    private Double unitPrice;
    private Integer quantity;

}

File: order_processing/src/model/article.go
package model

type Article struct {
    ID     		int64  	`json:"id"`
    Code        string  `json:"code"`
    Name  		string  `json:"name"`
    UnitPrice 	float64 `json:"unitPrice"`
    Quantity  	int64 	`json:"quantity"`
    Subtotal    float64 `json:"subtotal"`
}